addons/marketing_card/models/mailing_mailing.py

```diff
@@ -71,8 +72,8 @@ def action_update_cards(self):
 
     def _get_recipients_domain(self):
         """Domain with an additional condition that the card must exist for the records."""
-        domain = super()._get_recipients_domain()
+        domain = Domain(super()._get_recipients_domain())
```

Comment:
Since its `super()` in our codes has already return `Domain`, maybe we can remove the `Doamin(` here.

Reply:
Done.